File: C - language/Q17.c
#include <stdio.h>

int main() {
    int arr[]={1,-2,3,-4,5,-6};
    int len=sizeof(arr)/sizeof(arr[0]);
    int count;
    int i;
    for (i=0;i<len;i++){
        if(arr[i]<0){
            count++;
        }
    }
    printf("Positve:%d\n",len-count);
    printf("Negative:%d",count);

    return 0;
};

File: javascript/Q17.js
var arr = [1,-2,3,-4,5,-6];
var neg=0;
var pos=0;

for (i=0;i<arr.length;i++){
    if (arr[i]<0){
        neg++
    }else{
        pos++
    }
}

console.log("Positive:",pos,"   Negative:",neg)